dockerfiles/Dockerfile

```diff
@@ -69,44 +65,31 @@ RUN git clone https://github.com/robotology/robotology-superbuild.git --depth 1
           -DROBOTOLOGY_USES_GAZEBO:BOOL=ON \
           -DROBOTOLOGY_ENABLE_DYNAMICS:BOOL=ON \
           -DROBOTOLOGY_ENABLE_ROBOT_TESTING:BOOL=ON && make
-#    && cd ../.. && rm -Rf robotology-superbuild
 
 # Download and build fork of icub-main
 RUN cd /robotology-superbuild/src/ICUB && \
     git remote add upstream https://github.com/mfussi66/icub-main.git && \
-    git pull upstream master
-
-RUN cd /robotology-superbuild/build/ && make
+    git pull upstream master && \
+    cd /robotology-superbuild/build/ && \
+    make
 
 # Clean up git configuration
 RUN git config --global --unset-all user.name && \
     git config --global --unset-all user.email
 
+# Build package containing forward kinematics inspection
+RUN git clone https://github.com/icub-tech-iit/icub-forward-kinematics-inspection.git && \
+    cd icub-forward-kinematics-inspection && mkdir build && cd build && \
+    source ${ROBOTOLOGY_SUPERBUILD_INSTALL_DIR}/share/robotology-superbuild/setup.sh && \
```

Comment:
`source ${ROBOTOLOGY_SUPERBUILD_INSTALL_DIR}/share/robotology-superbuild/setup.sh` this should let you avoid specifying `iDynTree_DIR`.

Reply:
the build process fails if it is written that way. It is necessary to run the source command as `/bin/bash -c "source .../.../setup.bash`. 

I corrected it and continued the build, but vnc is still not available on the browser.

Reply:
Good catch 👍🏻 
I might have some time tomorrow morning to try it out and see myself.

Reply:
```suggestion
    sh -c 'source ${ROBOTOLOGY_SUPERBUILD_INSTALL_DIR}/share/robotology-superbuild/setup.sh' && \
```